.swiftlint.yml

```diff
@@ -1,4 +1,5 @@
 disabled_rules:
+  - closure_parameter_position
```

Comment:
adding this just to appease swiftlint for now, but I think it's a good rule. we should take a separate pass to fix the dozen or so violations later.